examples/example_plugin/example_plugin/urls.py

```diff
@@ -94,4 +94,5 @@
         views.DeviceDetailPluginTabTwoView.as_view(),
         name="device_detail_tab_2",
     ),
+    path("override-target/", views.ViewToBeOverridden.as_view(), name="view_to_be_overridden"),
```

Comment:
Would be good to add an explanatory comment here documenting why this view/URL pattern exists.